### ISSUE TITLE
Bug: Square Your Sets in context menu in SD does not work

### ISSUE THREAD

**Issue by mpogue2** [mpogue2/SquareDesk#534]
If I type in "square your sets", works fine.  But, right click on sequence, and select Square your Sets, and it doesn't do anything.

**Comment by mpogue2** [comment]
Context menu prints this on the console:

```
QObject::connect: No such slot MainWindow::square_your_sets()
QObject::connect:  (receiver name: 'MainWindow')

```

**Comment by mpogue2** [comment]
Fixed by 069dc5b7c98130368a5483e26ab8b865331bef4b .
